File: FinanceApp.Shared/Entities/UserTables/CreditCard.cs
namespace FinanceApp.Shared.Entities.UserTables
{
    public class CreditCard : UserTable
    {
        public string Name { get; set; } = String.Empty;
        public int InvoiceClosingDay { get; set; }
        public int InvoicePaymentDay { get; set; }

        public override void CheckInput()
        {
        }
    }
}

File: FlutterApp/finance_app/lib/models/credit_card/credit_card.dart
class CreditCard {
  int id;
  String name;
  int invoiceClosingDay;
  int invoicePaymentDay;

  CreditCard(
      {required this.id,
      required this.name,
      required this.invoiceClosingDay,
      required this.invoicePaymentDay});

  CreditCard.fromJson(Map<String, dynamic> json)
      : id = json['id'],
        name = json['name'],
        invoiceClosingDay = json['invoiceClosingDay'],
        invoicePaymentDay = json['invoicePaymentDay'];
}
